crates/youki/src/observability.rs

```diff
@@ -73,7 +73,13 @@ where
     let log_level_filter = tracing_subscriber::filter::LevelFilter::from(level);
     let log_format = detect_log_format(config.log_format.as_deref())
         .with_context(|| "failed to detect log format")?;
-    let systemd_journald = if config.systemd_log {
+
+    #[cfg(debug_assertions)]
+    let journald = true;
+    #[cfg(not(debug_assertions))]
+    let journald = config.systemd_log;
+
+    let systemd_journald = if journald {
```

Comment:
@yihuaf May I ask you to review around there?

Reply:
This should be moved in an issue to track. I need to understand how conmon is handling the logs.

Reply:
https://github.com/containers/youki/issues/2281

Reply:
Thanks :+1: I'll mark this comment as resolved for now